packages/plugin-compiler/src/main/kotlin/io/realm/compiler/AccessorModifierIrGeneration.kt

```diff
@@ -479,10 +481,13 @@ class AccessorModifierIrGeneration(private val pluginContext: IrPluginContext) {
                                     // For managed property call C-Interop function
                                     cinteropCall,
                                     // For unmanaged property set backing field
-                                    irSetField(
-                                        irGet(receiver),
-                                        backingField,
-                                        irGet(setter.valueParameters.first())
+                                    IrSetFieldImpl(
```

Comment:
Do we need to use the `IrSetFieldImpl` here instead of the `irSetField`. It feels like we are accessing something we perhaps shouldn't? 🤔

Reply:
`irSetField` [signature](https://github.com/JetBrains/kotlin/blob/dump-model-compiler-key-1.6.10/compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/builders/ExpressionHelpers.kt#L144) changed in [1.6.20](https://github.com/JetBrains/kotlin/blob/1.6.20/compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/builders/ExpressionHelpers.kt#L146) The only workaround that will avoid us to bump to `1.6.20` to use the new signature is to call `IrSetFieldImpl` directly